test-framework/arquillian/pom.xml

```diff
@@ -0,0 +1,74 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- ~ Copyright 2018 Red Hat, Inc. ~ ~ Licensed under the Apache License, 
```

Comment:
Not sure we need copyright headers in `pom.xml`, plus the formatting is not right

Reply:
We do have headers in other poms but you're right that the formatting is not right.